win32ss/user/ntuser/simplecall.c

```diff
@@ -372,99 +426,120 @@ NtUserCallOneParam(
 DWORD_PTR
 APIENTRY
 NtUserCallTwoParam(
-   DWORD_PTR Param1,
-   DWORD_PTR Param2,
-   DWORD Routine)
+    DWORD_PTR Param1,
+    DWORD_PTR Param2,
+    DWORD Routine)
 {
-   PWND Window;
-   DECLARE_RETURN(DWORD_PTR);
-
-   TRACE("Enter NtUserCallTwoParam\n");
-   UserEnterExclusive();
-
-   switch(Routine)
-   {
-      case TWOPARAM_ROUTINE_REDRAWTITLE:
-         {
-           DWORD_PTR Ret;
-           Window = UserGetWindowObject((HWND)Param1);
-           Ret = (DWORD_PTR)UserPaintCaption(Window, (INT)Param2);
-           RETURN(Ret);
-         }
-      case TWOPARAM_ROUTINE_SETMENUBARHEIGHT:
-         {
+    PWND Window;
+    DWORD_PTR Ret;
+    TRACE("Enter NtUserCallTwoParam\n");
+    UserEnterExclusive();
+
+    switch (Routine)
+    {
+        case TWOPARAM_ROUTINE_REDRAWTITLE:
+        {
             DWORD_PTR Ret;
```

Comment:
Shadowing variable. Surprised it doesn't give you an uninitialized warning for the outer one.

Reply:
Fixed